modules/database/src/adapters/sequelize-adapter/parser/index.ts

```diff
@@ -93,23 +96,48 @@ function _parseQuery(
     if (key === '$or') {
       Object.assign(parsed, {
         [Op.or]: query[key].map((operation: ParsedQuery) =>
-          _parseQuery(operation, relations, associations),
+          _parseQuery(operation, dialect, relations, associations),
         ),
       });
     } else if (key === '$and') {
       Object.assign(parsed, {
         [Op.and]: query[key].map((operation: ParsedQuery) =>
-          _parseQuery(operation, relations, associations),
+          _parseQuery(operation, dialect, relations, associations),
         ),
       });
     } else if (key === '$regex') {
       Object.assign(parsed, { [Op.regexp]: query[key] });
     } else if (key === '$options') {
       continue;
     } else {
-      const subQuery = _parseQuery(query[key], relations, associations);
+      if (!!query[key] && typeof query[key] === 'object' && !Array.isArray(query[key])) {
+        const likeCandidates = Object.keys(query[key]);
+        if (likeCandidates.includes('$like')) {
```

Comment:
are we expecting the "%" to be applied by the devs here?

Reply:
Yes. Custom endpoints automatically wrap the provided string so as to remain backwards compatible.
Direct query users should include percentage signs in their `$like` / `$ilike` operators.